src/options-from-args.js

```diff
@@ -0,0 +1,50 @@
+// @flow
+import type {Options} from "./types.js";
+import type {minimistOutput} from "minimist";
+
+/**
+ * Convert arguments to options.
+ */
+export const optionsFromArgs = (args: minimistOutput): $Shape<Options> => {
+    const options: $Shape<Options> = {};
+    if (args._ && args._.length > 0) {
+        options.includeGlobs = args._;
+    }
+
+    const excludeGlobs = (args.ignore: any)?.split(";").filter((c) => !!c);
+    if (excludeGlobs != null) {
+        options.excludeGlobs = excludeGlobs;
+    }
```

Comment:
Can `ignore` be `false` like `ignoreFiles`?

Reply:
No, it doesn't work that way - it's either a string, or its not provided. The main reason for the `?` is to work around the ambiguity of minimist typing and the fact that this argument may not be supplied.